src/style.css

```diff
@@ -1,3 +1,80 @@
+:root {
+  --colors-link: rgba(0, 172, 220, 1);
+}
+
 body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, Helvetica, serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+}
+
+header {
+  width: 100vw;
+  display: flex;
+  position: relative;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
+}
+
+.logo {
+  margin-left: 50px;
+}
+
+nav {
+  display: flex;
+  position: relative;
+  margin: 0;
+  right: 50px;
+}
+
+ul {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  list-style: none;
+}
+
+.aa {
+  display: inline-block;
+  text-decoration: none;
+  color: black;
```

Comment:
The color for the `.aa` class is set to black. Ensure this provides enough contrast against the background for accessibility. Consider using a tool to check color contrast ratios.